rviz_default_plugins/test/rviz_default_plugins/displays/image/ros_image_texture_test.cpp

```diff
@@ -33,7 +33,14 @@
 #include <memory>
 #include <vector>
 
-#include <Ogre.h> // NOLINT
+#ifdef _WIN32
+# pragma warning(push)
+# pragma warning(disable : 4996)
+#  include <Ogre.h>  // NOLINT
+# pragma warning(pop)
+#else
+# include <Ogre.h>  // NOLINT
```

Comment:
I would prefer that we do not repeat the include statement in this case since it's not different across OS's.

Reply:
Here's what I would prefer:

```c++
#ifdef _WIN32
# pragma warning(push)
# pragma warning(disable : 4996)
#endif

#include <Ogre.h>  // NOLINT

#ifdef _WIN32
# pragma warning(pop)
#endif
```

I tried to make pr through the webui, but it gave me a 404 for some reason.

Is this change ok with you @greimela-si?

Reply:
Yes, that looks good to me!